control/joystick_commander/include/can_read.h

```diff
@@ -0,0 +1,57 @@
+/************************************************************************/
+/* The MIT License (MIT) */
+/* ===================== */
+
+/* Copyright (c) 2016 PolySync Technologies, Inc.  All Rights Reserved. */
```

Comment:
Need to update our copyrights, but we have a tool that can sweep all files - we can get that in another commit.